<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatRoom</class>
 <widget class="QWidget" name="ChatRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>聊天室</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>创建房间</string>
     </property>
    </widget>
   </item>
   <item row="6" column="6">
    <widget class="QPushButton" name="inputbtn">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>200</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 0);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" rowspan="3" colspan="4">
    <widget class="QTextEdit" name="techat">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/image/chengyuanlist.png);

background-color: rgb(0, 174, 237);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 174, 237);
image: url(:/new/prefix1/image/liaotianlist.png);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QListWidget" name="lwUser">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QListWidget" name="chatroom">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
gridline-color: rgb(170, 255, 255);
selection-background-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2" colspan="6">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="quitbtn">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 255, 255);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/new/prefix1/image/quit.png</normaloff>
         <normalon>:/new/prefix1/image/quit.png</normalon>
         <selectedoff>:/new/prefix1/image/quit.png</selectedoff>
         <selectedon>:/new/prefix1/image/quit.png</selectedon>:/new/prefix1/image/quit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 255, 255);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>充值</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string> 余额 ¥</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 255, 255);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/new/prefix1/image/刷新.png</normaloff>:/new/prefix1/image/刷新.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rocket">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 255, 255);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>火箭</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="flower">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 255, 255);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>飞机</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="good">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 255, 255);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>轮船</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="FbBtn">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 255, 255);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>鱼翅</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="3" colspan="3">
    <widget class="QLineEdit" name="leinput">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="startinput">
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>开启直播</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>startinput</zorder>
  <zorder>inputbtn</zorder>
  <zorder>label_2</zorder>
  <zorder>leinput</zorder>
  <zorder>label</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>lwUser</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>chatroom</zorder>
  <zorder>techat</zorder>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
